admin.go

```diff
@@ -0,0 +1,360 @@
+package sarama
+
+import "errors"
+
+// ClusterAdmin is the administrative client for Kafka, which supports managing and inspecting topics,
+// brokers, configurations and ACLs. The minimum broker version required is 0.10.0.0.
+// Methods with stricter requirements will specify the minimum broker version required.
+// This client was introduced in 0.11.0.0 and the API is still evolving.
```

Comment:
I don't think this line is necessary in the documentation?

Reply:
ok. will remove that . I was thinking mentioning the version dependency would be needed especially admin related stuff is supported only in latest versions of kafka.

Reply:
removed it